### PR TITLE
feat: use native async generators/iterators

### PR DESCRIPTION
**Pull request by jquense**
maybe not ready for this downstream but here we go…

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ cache: 'yarn'
 notifications:
   email: false
 node_js:
-  - 'stable'
+  - '10'
```
Comment (taion) [inline .travis.yml]: is "stable" not 10?

File: examples/todo/package.json [hunk 2]
```diff
@@ -25,7 +26,7 @@
     "jwks-rsa": "^1.2.1",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
-    "react-relay": "^1.4.1",
+    "react-relay": "~1.4.1",
```
Comment (taion) [inline examples/todo/package.json]: ~1.5.0

File: .travis.yml (hunk 1 above)
Comment (jquense) [reply .travis.yml]: i guess it is haha

**Pull request merged**